component documentation/componentDoc.css

```diff
@@ -0,0 +1,190 @@
+:root {
+    --primary-color: white ;
+    --background-color: rgb(29, 27, 27);
+    --secondary-color: #6970e7;
+    --text-color: #737477;
+    --backgroundTwo-color: rgb(34, 32, 32); 
```

Comment:
use shade name instead of `--backgroundTwo-color`

Reply:
> use shade name instead of `--backgroundTwo-color`
thank you, I will change that